### PR TITLE
Annual report 2022

### PR DESCRIPTION
**Pull request by ElenaFdR**
This is a PR for the annual report in Markdown, but not the MD financial report, which still has some formatting to reconcile. Since I've pulled the financial report out of this branch, I also propose squashing and merging this PR.

### CODE REVIEW

**Comment by ElenaFdR** [comment]
Re failing the GitHub Actions build: 
1. we've used bold for emphasis in both previous years, as well as in other places in the doc where it hasn't been flagged. Can we be flexible for this? Why has GH Actions been inconsistent? 
2. I don't see the trailing space in my local version, so this is hard to fix.

**Review by ericherman — APPROVED** [review]

File: organization/annual-reports/annual-report-2022.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+Board members do not receive any compensation, and did not incur any expenses for their activities.
+
+Though we planned to recruit additional board members in 2022, we delayed until we had more clarity about the Foundation’s future legal structure, business model and governance - we didn’t feel comfortable asking new board members to join with such a vaguely defined set of responsibilities. We hope to welcome new board members in 2023 who are ready to lead us through our next major transition.
+
+### Our finances
+
+This section provides an overview of the detailed financial report that accompanies this annual report.
+
+[See the full financial report](/financial-report-2022.md).
```
Comment (Ainali) [inline organization/annual-reports/annual-report-2022.md:173]: This file is not part of the commit yet.
